### DISCUSSION TITLE
[功能建议] 禁止用户自己审批自己提交的工单

### DISCUSSION THREAD

**Discussion by delphid** [hhyo/Archery#2039] [Ideas]
### 目前遇到的问题/使用障碍
如果工单审批权限放的比较宽的话，可能导致大部分用户可以自己审批自己提交的上线工单，让审批功能失去效果。

### 希望如何解决/实现它
希望能禁止用户自己审批自己提交的工单（不过可以豁免超级用户，便于调试）。可以考虑做成一个配置项。

### 具体实现提议
https://github.com/delphid/Archery/commit/03c848bf9af73e55135b8ac529fcdba34ccb5574

**Comment by LeoQuote** [comment]
欢迎提交 pr
